main.tf

```diff
@@ -339,7 +339,7 @@ https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-iam-roles.html
 resource "aws_iam_role" "ec2_autoscaling" {
   count              = var.enabled ? 1 : 0
   name               = module.label_ec2_autoscaling.id
-  assume_role_policy = join("", data.aws_iam_policy_document.assume_role_ec2.*.json)
+  assume_role_policy = join("", data.aws_iam_policy_document.assume_role_emr.*.json)
```

Comment:
nice find, thanks @navdeep710

Reply:
thanks @aknysh, this module is some sweet ops :)